tests/test_prescan_sub.py

```diff
@@ -400,8 +393,8 @@ def test_bias_hvoff():
             
             output_dataset = prescan_biassub(dataset, return_full_frame=return_full_frame)
 
-            # Compare median bias measurement to expectation
-            if np.abs(output_dataset[0].ext_hdr['MED_BIAS'] - bval) > tol:
+            # Compare bias measurement to expectation
+            if np.any(np.abs(output_dataset[0].bias - bval) > tol):
```

Comment:
Don't need to change this now, as it is clear what you are doing here, but `bias == pytest.approx(bval, tol)` is a thing you can use in the future.